spaces/domains.json

```
@@ -285,5 +285,6 @@
   "vote.to-the-moon.farm": "to-the-moon-farm.eth",
   "vote.polyvertex.finance": "polyvertex.eth",
   "gov.aol.gg": "apesonline.eth",
-  "gov.coin-bid.io": "coin-bid.eth"
+  "gov.coin-bid.io": "coin-bid.eth",
+  "vote.stablegaj.finance", "stablegaj.eth",
```

Hi Gajesh! Great to see you here :). There is a small issue here, the format should be like this:
```suggestion
  "vote.stablegaj.finance": "stablegaj.eth"
```